arkouda/pdarrayclass.py

```diff
@@ -1168,6 +1188,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list[str]:
+    """
+    Return a list containing the names of all registered objects
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the registry
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    registered_list: list[str] = []
+
+    if info('__RegisteredSymbols__') == '__EMPTY_REGISTRY__':
+        return registered_list
+
+    registered_object: str
+    for registered_object in filter(None, info('__RegisteredSymbols__').split('\n')):
+        if registered_object is not None:
+            name = registered_object.split()[0].split(':')[1].replace('"', '')
```

Comment:
No change required,  stylistic comment:
It may seem silly for a one-liner, but moving this object parsing to its own function makes it more test-able from a unit testing perspective and easier to swap out later... although it would add a bit more complexity (additional method etc.) to there is a trade-off.

I know we've been discussing moving the return structure to some type of json so it's probably best to wait for that to happen.